src/porepy/numerics/fv/mpfa.py

```diff
@@ -72,461 +69,260 @@ def discretize(self, g, data):
-            scipy.sparse.csr_matrix (shape num_faces, num_cells): flux
-                discretization, in the form of mapping from cell pressures to face
-                fluxes.
-            scipy.sparse.csr_matrix (shape num_faces, num_faces): discretization of
-                boundary conditions. Interpreted as fluxes induced by the boundary
-                condition (both Dirichlet and Neumann). For Neumann, this will be
-                the prescribed flux over the boundary face, and possibly fluxes
-                over faces having nodes on the boundary. For Dirichlet, the values
-                will be fluxes induced by the prescribed pressure. Incorporation as
-                a right hand side in linear system by multiplication with
-                divergence operator.
-            scipy.sparse.csr_matrix (shape num_faces, num_cells): Used to recover
-                pressure on boundary faces. Contribution from computed cell
-                pressures only; contribution from faces (below) also needed.
-            scipy.sparse.csr_matrix (shape num_faces, num_faces): Used to recover
-                pressure on boundary faces. Contribution from boundary conditions.
-            scipy.sparse.csr_matrix (shape num_faces, num_cells*dim): discretization of
-                the flux due to vector source term, in the form of mapping from cell vector
-                sources to face fluxes. Only if vector_source is True
-        Example:
-            # Set up a Cartesian grid
-            g = structured.CartGrid([5, 5])
-            k = tensor.SecondOrderTensor(g.dim, np.ones(g.num_cells))
-            g.compute_geometry()
-            # Dirirchlet boundary conditions
-            bound_faces = g.tags['domain_boundary_faces'].ravel()
-            bnd = bc.BoundaryCondition(g, bound_faces, ['dir'] * bound_faces.size)
-            # Discretization
-            flux, bound_flux, bp_cell, bp_face = mpfa(g, k, bnd)
-            # Source in the middle of the domain
-            q = np.zeros(g.num_cells)
-            q[12] = 1
-            # Divergence operator for the grid
-            div = fvutils.scalar_divergence(g)
-            # Discretization matrix
-            A = div * flux
-            # Assign boundary values to all faces on the bounary
-            bound_vals = np.zeros(g.num_faces)
-            bound_vals[bound_faces] = np.arange(bound_faces.size)
-            # Assemble the right hand side and solve
-            rhs = q + div * bound_flux * bound_vals
-            x = sps.linalg.spsolve(A, rhs)
-            # Recover flux
-            f = flux * x - bound_flux * bound_vals
-            # Recover boundary pressure
-            bp = bp_cell * x + bp_face * bound_vals
-        """
+        # NOTE: active_faces are all faces to have their stencils updated, while
+        # active_cells may form a larger set (to accurately update all faces on a
+        # subgrid, it is necessary to assign some overlap in terms cells).
```

Comment:
I'm not entirely sure what "in terms of" means here, but I think there should be an "of"
```suggestion
        # subgrid, it is necessary to assign some overlap in terms of cells).
```